src/test/txdb_tests.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2019-2020 Chaintope Inc.
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <txdb.h>
+#include <test/test_tapyrus.h>
+
+#include <boost/test/unit_test.hpp>
+
+BOOST_FIXTURE_TEST_SUITE(txdb_tests, BasicTestingSetup)
+
+BOOST_AUTO_TEST_CASE(txdb_aggpubkey_serialize_tests)
+{
+    XFieldAggpubkey xFieldAggpubkey;
+    std::vector<XFieldAggpubkey> xFieldAggpubkeyList;
+    CDataStream stream(ParseHex("0121025700236c2890233592fcef262f4520d22af9160e3d9705855140eb2aa06c35d30a0000000000000000000000000000000000000000000000000000000000000000000000"), SER_NETWORK, PROTOCOL_VERSION);
+    stream >> xFieldAggpubkeyList;
+
+    // 1 item
+    xFieldAggpubkey = xFieldAggpubkeyList[0];
+    BOOST_CHECK_EQUAL(HexStr(xFieldAggpubkey.aggpubkey.begin(), xFieldAggpubkey.aggpubkey.begin()+xFieldAggpubkey.aggpubkey.size()), "025700236c2890233592fcef262f4520d22af9160e3d9705855140eb2aa06c35d3");
+    BOOST_CHECK_EQUAL(xFieldAggpubkey.height, 10);
+    BOOST_CHECK_EQUAL(xFieldAggpubkey.blockHash, uint256S("0000000000000000000000000000000000000000000000000000000000000000000000"));
```

Comment:
`0000000000000000000000000000000000000000000000000000000000000000000000` value is 35 bytes, `0000000000000000000000000000000000000000000000000000000000000000` is correct.

Reply:
Sorry! that was copied from the stream. But when the length is more there is no error!